app/Actions/Jetstream/UserProfile.php

```diff
@@ -16,12 +16,11 @@ class UserProfile
      */
     public function __invoke(Request $request, array $data): array
     {
-        $providers = collect(config('auth.login_providers'))->filter(fn ($provider) => ! empty($provider));
-        $providersName = $providers->mapWithKeys(function ($provider) {
-            return [
-                $provider => config("services.$provider.name") ?? __("auth.login_provider_{$provider}"),
-            ];
-        });
+        $providers = collect(config('auth.login_providers'))->filter(fn ($provider) => ! empty($provider))
```

Comment:
This is really hard to read, in my opinion. So it's a collection that we filter, and if the provider is not empty, we map each one of these into an array that contains two ternary operators that do something at the end.... Can you make this simpler to read?